sdk/java-sdk-spring/docker-compose-integration.yml

```diff
@@ -28,4 +31,18 @@ services:
       test: curl http://0.0.0.0:8085/v1/projects/test/topics || exit -1
       interval: 5s
       timeout: 5s
-      retries: 10
\ No newline at end of file
+      retries: 10
+
+  tempo:
+    image: grafana/tempo:latest
+    command: [ "-config.file=/etc/tempo.yaml" ]
+    volumes:
+      - ./tempo.yaml:/etc/tempo.yaml
+      - ./tempo-data:/tmp/tempo
+    ports:
+      - "14268:14268"  # jaeger ingest
+      - "3200:3200"   # tempo
+      - "9095:9095" # tempo grpc
+      - "4317:4317"  # otlp grpc
+      - "4318:4318"  # otlp http
+      - "9411:9411"   # zipkin
```

Comment:
Ok, so the plan is to always run a server. Does that work? What about integration tests/test containers?

It also means the docker compose with both will be needed for all samples (and the g8 template).

Reply:
This change is made exclusively for the integration tests/testContainer. See [this](https://github.com/lightbend/kalix-jvm-sdk/pull/1763/files#diff-7c37fa3ed1fba565ba77f927e8f7f8d82e92bc274d54130ef8d2ed4a72ade74f), please.

Not sure that I follow that "the plan is to always run a server". If the proxy telemetry.tracing is disabled (which is by default) no traces should be created nor pushed.